server/src/com/cloud/api/ApiResponseHelper.java

```diff
@@ -479,6 +479,17 @@ public SnapshotResponse createSnapshotResponse(Snapshot snapshot) {
             if (zone != null) {
                 snapshotResponse.setZoneId(zone.getUuid());
             }
+
+            if (volume.getVolumeType().equals(Volume.Type.ROOT)) {
```

Comment:
Enumerated values are guaranteed to be singletons.   Therefore, the ``==`` operator deterministically performs both reference and value equality checks.  It is also preferred because it avoid an NPE in the event that ``volume.getVolumeType()`` returns null.